bin/nuxt-component-meta.mjs

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+import { runGenerate } from '../dist/runtime/cli/index.mjs'
```

Comment:
CLI entrypoint should not be in `dist/runtime` (it is a build time entry)

Reply:
@pi0, I think you reviewed an old commit, I moved it to `'../dist/cli.mjs'` using a new build.entries in package.json

Reply:
Do you think it's relevant to have a nuxt cli kit that would export those kinds of utils: https://github.com/nuxtlabs/nuxt-component-meta/pull/60/files#diff-280cd3ea2fd76f9dc66e2878205eed73fd8006764453fee4c1f08742d4840f2d

Reply:
Yes i was thinking CLI could expose kit/nuxt loading utils also had other usecases 👍🏼